__tests__/app.test.js

```diff
@@ -47,40 +47,61 @@ describe("GET /api/", () => {
+                article_id: expect.any(Number),
                 title: expect.any(String),
                 topic: expect.any(String),
                 author: expect.any(String),
                 body: expect.any(String),
                 created_at: expect.any(String),
                 votes: expect.any(Number),
-                article_img_url:expect.any(String)
-              }
-            expect(status).toBe(200)
-            expect(article).toMatchObject(expectedArticle)
-        })
-        test("404: status code when article not found",async () => {
-            const { status, body } = await request(app).get("/api/article/99999")
-            const {message} = body
+                article_img_url: expect.any(String)
+            };
+            expect(status).toBe(200);
+            expect(article).toMatchObject(expectedArticle);
+        });
+        test("404: status code when article not found", async () => {
+            const { status, body } = await request(app).get("/api/article/99999");
+            const { message } = body;
 
-            expect(status).toBe(404)
-            expect(message).toBe("Article not found")
+            expect(status).toBe(404);
+            expect(message).toBe("Article not found");
 
 
-             
-        })
-        test("400: status code when giving invalid type of article_id",async() => {
-             const {status,body} = await request(app).get("/api/article/banana")
-             const {message} = body
 
-             expect(status).toBe(400)
-             expect(message).toBe("banana is an invalid article_id (number)")
+        });
+        test("400: status code when giving invalid type of article_id", async () => {
+            const { status, body } = await request(app).get("/api/article/banana");
+            const { message } = body;
+
+            expect(status).toBe(400);
+            expect(message).toBe("banana is an invalid article_id (number)");
+        });
+    });
+    describe.only("/articles", () => {
+        test("200:status code and responds with article array of article objects and corresponding properties", async () => {
+            const { status, body } = await request(app).get("/api/articles");
+            const { articles } = body;
+            const expectedArticleProperties = ["author","title","article_id","topic","created_at",
+                "votes","article_img_url","comment_count"]
+            
+            expect(status).toBe(200)
+            expect(articles.length).not.toBe(0)
+            articles.forEach((article) => {
+                expect(Object.keys(article)).toEqual(expectedArticleProperties)
+            })
+        
+        });
+        test("200: articles are sorted by created_at date in descending order",async() => {
+            const { body } = await request(app).get("/api/articles")
+            const { articles } = body;
+
+             expect(articles).toBeSortedBy("created_at",{descending:true})
```

Comment:
Good test - you could merge this into the test above instead of making two requests